packages/plugin-ext/src/plugin/types-impl.ts

```diff
@@ -2009,3 +2009,36 @@ export enum WebviewPanelTargetArea {
     Right = 'right',
     Bottom = 'bottom'
 }
+export class CallHierarchyItem {
```

Comment:
I hope this types are just copied without any modifications from VS Code codebase.

Reply:
Yes, the code was in `@theia/plugin-ext` before (copied from VS Code)

Reply:
I hope to be able to put this file next to counterpart to VS Code and see differences like explained in this PR: https://github.com/eclipse-theia/theia/pull/6903/files#diff-905508836c18f47e2fc9e75198b465e8R21-R32

I would appreciate if we just keep types and don't change anything to simplify synching.